tests/codex/test-codex-skills.sh

```
@@ -90,6 +98,36 @@ else
   fail ".codex/README.md not found"
 fi
 
+# 6b. README documents Codex skill installation and the new review skill
+if grep -Fq '~/.codex/skills' "$ROOT/.codex/README.md"; then
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)